test/clj/rems/test_browser.clj

```diff
@@ -1264,4 +1331,6 @@
                   "Active" true
                   "Last modified" (get-organization-last-modified (btu/context-get :organization-id))
                   "Modifier" "Organization Owner 2 (organization-owner2@example.com)"}
-                 (slurp-fields :organization))))))))
+                 (slurp-fields :organization)))))))
+
+  "Organization Owner 1 (organization-owner1@example.com)\nOrganization Owner 2 (organization-owner2@example.com)")
```

Comment:
this can't be doing anything or am I missing somehting? it's just a string expression on its own?

Reply:
True, removed.